.github/actions/build/action.yml

```diff
@@ -0,0 +1,36 @@
+name: Build
+
+runs:
+  using: "composite"
+  steps:
+    - name: Configure rustup
+      id: rustup
+      run: |
+        rustup update
+        rustup toolchain install --profile minimal nightly
+        rustup target add --toolchain nightly x86_64-pc-windows-msvc
+      shell: bash
```

Comment:
IIRC the default value of `shell` on the ubuntu runner is `bash` so we don't need this.

Reply:
It does feel redundant but apparently in actions you must specify the shell, wouldn't run without it

Reply:
Ah, so a template-like action needs it? Then totally fine.